1 The Physical/Guess Point Level 1 (Stateless)/1 - Red-Green-Refactor Practice/1.1 - Fizzbuzz/src/fizzbuzz.ts

```diff
@@ -1,3 +1,9 @@
 export function fizzbuzz(number: number) {
-  return 'Fizz';
+  if (number % 3 === 0) {

```

Comment:
Great work so far !  What I would suggest you is this:
Make sure you follow these steps: 
- Fake it
- Obvious Implementation
- Triangulation
- Try to make the code as readable as possible while being at REFACTOR (third) step.
So what I would do there is I would not jump directly to the right implementation but I would try to reach it by using the rule of three.
e.g:
if number === 3 ? return fizz
if number === 6 ? return fizz
if number === 9 ? return fizz
Now we can apply the rule of three and make it obvious
if number % 3 === 0 ? return fizz
Hopefully it makes sense. Please let me know what do you think.

Reply:
yep, got it. Thanks!